src/datafactory/azext_datafactory/manual/custom.py

```diff
@@ -11,30 +11,45 @@
 from knack.util import CLIError
 
 
-def datafactory_factory_create(client,
-                               resource_group_name,
-                               factory_name,
-                               if_match=None,
-                               location=None,
-                               tags=None,
-                               factory_vsts_configuration=None,
-                               factory_git_hub_configuration=None,
-                               global_parameters=None):
+def datafactory_create(client,
```

Comment:
wondering why do we need to overide this? looks the same as the generated code ?

Reply:
In generated code, identity is empty: `factory['identity'] = {}`. Portal behavior is enabling system assigned identity by default, the change aligns the behavior with Portal.

Reply:
Maybe can try to set default value for identity ?

Reply:
and hide this parameter with directive

Reply:
The parameter is already hidden. The default value is not a str, but an object, is it able to do so?